package.json

```diff
@@ -24,9 +24,9 @@
     ]
   },
   "dependencies": {
-    "@expo/vector-icons": "10.1.0",
-    "@react-native-community/datetimepicker": "2.3.2",
-    "@react-native-community/masked-view": "0.1.6",
+    "@expo/vector-icons": "^10.0.0",
```

Comment:
Please do not put downgrade any package.

Reply:
I've restored the original version of this dependency. But I've downgraded one other dependency due to the reason stated in the comment below